packages/react/src/components/Authenticator/Router/index.tsx

```diff
@@ -35,39 +35,41 @@ export function Router({ className, children }: RouterProps) {
         <View data-authenticator-variation="modal" />
 
         <View data-amplify-container="">
-          {(() => {
-            switch (route) {
-              case 'idle':
-                return null;
-              case 'confirmSignUp':
-                return <ConfirmSignUp />;
-              case 'confirmSignIn':
-                return <ConfirmSignIn />;
-              case 'setupTOTP':
-                return <SetupTOTP />;
-              case 'signIn':
-              case 'signUp':
-                return <SignInSignUpTabs />;
-              case 'forceNewPassword':
-                return <ForceNewPassword />;
-              case 'resetPassword':
-                return <ResetPassword />;
-              case 'confirmResetPassword':
-                return <ConfirmResetPassword />;
-              case 'verifyUser':
-                return <VerifyUser />;
-              case 'confirmVerifyUser':
-                return <ConfirmVerifyUser />;
+          <View data-amplify-body="">
```

Comment:
Added this in to fix styles, to match Angular and Vue implementation

Reply:
Is there a difference between `data-amplify-container` and `data-amplify-body`?  If not, we can rename the 1st one to match.

Reply:
the `data-amplify-container` is used all over the place. I'd say we leave it as is for now, but eventually we'll need to go through and clean up all the `data-` attributes

Reply:
`data-amplify-container` contains `data-amplify-body`, whereas `data-amplify-header` and `data-amplify-footer` contain the appropriate top-level Authenticator slots. This can be revisited though.